core/src/test/resources/log4j.properties

```diff
@@ -18,9 +18,20 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+log4j.appender.requestAppender=org.apache.log4j.varia.NullAppender
+
 log4j.logger.kafka=ERROR
 log4j.logger.org.apache.kafka=ERROR
```

Comment:
Should we be changing these 2 to `INFO`? It's unclear to me why we set them to `ERROR` by default. I also don't understand why the root logger is `OFF`.